src/fuzz/README.md

```diff
@@ -0,0 +1,119 @@
+# Fuzzing libspark
+
+## Quickstart Guide
+### Dependencies
```

Comment:
This is missing several `boost` and `openssl` dependencies that are required for the build to succeed.